wp-content/plugins/goonj-blocks/src/render.php

```diff
@@ -39,6 +41,14 @@
 	$action_target['id']
 );
 
+function format_date($date) {
+    return $date->format(DATE_FORMAT);
+}
+
+function format_time_range($start_date, $end_date) {
+    return $start_date->format(TIME_FORMAT) . ' - ' . $end_date->format(TIME_FORMAT);
+}
```

Comment:
let's create a new file `functions.php` inside this plugin and move these function there.

```suggestion
require_once __DIR__ . '/functions.php';
```

Reply:
sure